application/wso2verify/app/src/main/java/org/wso2/carbon/identity/mobile/wso2verify/BiometricAppConstants.kt

```diff
@@ -0,0 +1,15 @@
+package org.wso2.carbon.identity.mobile.wso2verify
+
+class BiometricAppConstants {
+    companion object {
+
+        const val POST = "POST"
+        const val CONTEXT_KEY = "sessionDataKey"
+        const val CHALLENGE = "challenge"
+        const val BODY = "body"
+        const val DOMAIN_NAME = "https://biometricauthenticator.private.wso2.com:9443"
```

Comment:
this should be taken during registration, please add a todo here, mentioning that